src/client/ABRPublishing.js

```diff
@@ -312,21 +355,35 @@ exports.CreateABRMezzanine = async function({
     });
 
     id = createResponse.id;
-    write_token = createResponse.write_token;
+    writeToken = createResponse.write_token;
   }
 
-  await this.CreateEncryptionConk({libraryId, objectId: id, writeToken: write_token, createKMSConk: true});
+  await this.CreateEncryptionConk({libraryId, objectId: id, writeToken, createKMSConk: true});
```

Comment:
I think this is a problem when using single master+mez objects.  We don't want to create a second content encryption key.